kotsadm/web/src/scss/components/forms/input.scss

```diff
@@ -17,6 +17,7 @@
     border: 1px solid #C4C7CA;
     padding: 6px 10px 7px;
     height: 30px;
+    width: 300px;
```

Comment:
This feels scary. Is this going to affect other inputs across the admin console?